### PR TITLE
Add Work details listing

### PR DESCRIPTION
**Pull request by jmnda-dev**
#10 
- [x] Add a table to show Work details.
- [x] Update to use `SWR` for data fetching.

Screenshot from 2021-11-17 21-20-14

### CODE REVIEW

File: src/pages/occupation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            </div>
+          </div>
+          <Pagination />
+        </div>
+      </div>
+    </>
+  );
+};
+
+export async function getStaticProps() {
```
Comment (sonlinux) [inline src/pages/occupation/index.tsx:84]: use  `SWR`  for every appi requests

**Review by sonlinux — CHANGES_REQUESTED** [review]
to the table listing do add beneficiary `Benenficiary` column to hold `first and last name` which should reference to  beneficiary details

File: src/pages/occupation/index.tsx (hunk 1 above)
Comment (jmnda-dev) [reply src/pages/occupation/index.tsx]: Will update

**Review by sonlinux — APPROVED** [review]
LGTM
Thanks @jmnda-dev

**Pull request merged**